landoapi/reviews.py

```diff
@@ -19,8 +19,10 @@
 
 
 def calculate_review_extra_state(
-    for_diff_phid, reviewer_status, reviewer_diff_phid, reviewer_voided_phid
-):
+    for_diff_phid: str,
+    reviewer_status: ReviewerStatus,
```

Comment:
nit: this is repeated in the docstring below, which can probably be removed/reworded below to reduce redundancy.